src/components/MapMenu.scss

```diff
@@ -5,3 +5,9 @@
   min-width: 200px;
   padding: .5rem 1rem;
 }
+
+.form-check {
+  padding-left: 8px !important;
+  padding-top: 2px !important;
+  padding-bottom: 2px !important;
```

Comment:
Can we achieve the same spacing using the `ps-*` and `py-*` classes and just get rid of this extra class?

Reply:
Tried to do that but apparently needed to add !important property to overwrite the default style.